src/main/java/frc/robot/commands/auto/AutoUtil.java

```diff
@@ -57,16 +58,17 @@ public static Pose2d getDrivePose(ParsedCommand parsedCommand) {
         double x_meters = AutoUtil.ParseDouble(parsedCommand.getArgument("x"), 0.0);
         double y_meters = AutoUtil.ParseDouble(parsedCommand.getArgument("y"), 0.0);
         double angle_degrees = AutoUtil.ParseDouble(parsedCommand.getArgument("angle"), 0.0);
-        return new Pose2d(x_meters, y_meters, Rotation2d.fromDegrees(angle_degrees));
+        var mirrorablePose = new FieldPose2024(new Pose2d(x_meters, y_meters, Rotation2d.fromDegrees(angle_degrees)));
+        return mirrorablePose.getCurrentAlliancePose();
```

Comment:
It's a good thing I found this since we'd only be testing autos for blue in the programming room! (Our red alliance autos would act like they were on the blue side of the field)